src/neptune/new/internal/operation_processors/async_operation_processor.py

```diff
@@ -51,15 +53,22 @@ def __init__(
         self,
         container_id: UniqueId,
         container_type: ContainerType,
-        queue: DiskQueue[Operation],
         backend: NeptuneBackend,
         lock: threading.RLock,
         sleep_time: float = 5,
         batch_size: int = 1000,
     ):
+        self._operation_storage = AsyncOperationStorage(container_id, container_type)
+
+        self._queue = DiskQueue(
+            dir_path=Path(self._operation_storage.data_path),
```

Comment:
Let's made `OperationStorage.data_path` as `Path` directly instead of returning a `str`